sdk/messages.go

```diff
@@ -69,6 +69,8 @@ var (
 	MsgWorkflowStarting                    = &Message{"MsgWorkflowStarting", trad{FR: "Le workflow %s#%s a été démarré", EN: "Workflow %s#%s has been started"}, nil}
 	MsgWorkflowError                       = &Message{"MsgWorkflowError", trad{FR: "Une erreur est survenue: %v", EN: "An error has occured: %v"}, nil}
 	MsgWorkflowNodeStop                    = &Message{"MsgWorkflowNodeStop", trad{FR: "Le pipeline a été arrété par %s", EN: "The pipeline has been stopped by %s"}, nil}
+	MsgWorkflowNodeMutex                   = &Message{"MsgWorkflowNodeMutex", trad{FR: "Le pipeline %s est en attente de la libération du mutex", EN: "The pipeline %s is waiting for mutex release"}, nil}
```

Comment:
Don't think that mutex is a good word for user experience. Maybe we have to say that this node is currently runnning on  another run

Reply:
I don't how to represent/call it on UI also ? Any idea ?

Reply:
"Ce pipeline ne peut être lancé tant qu'il est en cours sur un autre run. Mise en attente.."

Release:  "Lancement du pipeline"

No specific representation on UI, waiting status (with the clock icon) is enough i think.